modules/service/src/test/scala/ch/epfl/bluebrain/nexus/iam/service/routes/AclsRoutesSpec.scala

```diff
@@ -79,7 +79,7 @@ class AclsRoutesSpec extends AclsRoutesSpecInstances {
 
     "reject creating empty permissions" in {
```

Comment:
What about having some test that gets permissions on a path that does not exist for that identity? I don't find it covered here (even though I might be wrong).

Reply:
It'll come. I will have to cover such scenarios with exhaustive tests to implement #6.